test/cep_exact_data.jl

```diff
@@ -0,0 +1,30 @@
+using CEP
```

Comment:
the source code to calculate the exact_scenario
in case we ever change some of the input data.